src/main/java/school/lemon/changerequest/java/introduction/hw1/Task7.java

```diff
@@ -3,6 +3,42 @@
 
 public class Task7 {
     public static String getDayTitle(int dayIndex) {
-        return "";
+        String dayNumber;
+        switch (dayIndex)
+        {
+            case 1:
+                dayNumber = "Monday";
+                break;
+
+            case 2:
+                dayNumber = "Tuesday";
+                break;
+
+            case 3:
+                dayNumber = "Wednesday";
+                break;
+
+            case 4:
+                dayNumber = "Thursday";
+                break;
+
+            case 5:
+                dayNumber = "Friday";
+                break;
+
+            case 6:
+                dayNumber = "Saturday";
+                break;
+
+            case 7:
+                dayNumber = "Sunday";
+                break;
+
+            default:
+                dayNumber = "Error";
+                break;
+        }
+        return dayNumber;
```

Comment:
There's no need for `dayNumber` variable here. Just do `return` in each case clause.
Also, you should return `Error` in lowercase to pass the test.